robustness/loaders.py

```diff
@@ -60,8 +60,10 @@ def make_loaders(workers, batch_size, transforms, data_path, data_aug=True,
     else:
         if not only_val:
             train_set = custom_class(root=data_path, train=True, 
+                                        label_mapping=label_mapping,
                                         download=True, transform=transform_train)
         test_set = custom_class(root=data_path, train=False, 
+                                    label_mapping=label_mapping,
```

Comment:
Does this break loading other dataset with custom classes? It looks like it will which would be bad. If it does let me know because I already have a fix in mind.